tests/helpers.test.ts

```diff
@@ -2,7 +2,7 @@ import { expect } from 'chai';
 
 import { getCommonPathPrefix } from '../src/helpers';
 
-describe('helpers', () => {
+describe('helpers', function() {
```

Comment:
why this change? do you need `this`?

Reply:
[Mocha doesn't recommend using arrow functions](https://mochajs.org/#arrow-functions). I think it simpler to always use `function() {}` in tests than look at using `this` that belongs to Mocha test context